notebooks/timelapse-camera-tutorial.ipynb

```diff
@@ -41,145 +41,40 @@
     "* Cameras were installed on the east and west side of the Grand Mesa, across a vegetation scale of 1-9, using the convention __XMR__:\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.            #    image = Image.open(BytesIO(member))
</pre><p>Suggest not having any commented code in the final tutorial. Could this go in an admonition as an extra possible step?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SnowEx/snow-cookbook/pull/28/discussion/#comment-2471207731'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Forgot to remove that code - it was replaced by the code starting at Line 33.</p>